share/source.F90

```diff
@@ -37,7 +37,9 @@ function find_cell() result(icl_tmp)
     ! icl is now set
 
     if(icl_tmp .le. 0) then
-      write(*,*) 'Nonsense cell number stopping'
+       write(*,*) 'Nonsense cell number: ', icl_tmp
```

Comment:
As `(icl_tmp .le. 0)` is true, then `icl_tmp` is `-1`, which indicates that `icl_tmp` is not changed. We don't need to print `icl_tmp` here.

Reply:
Agreed